akka-cluster-sharding/src/test/scala/akka/cluster/sharding/PersistentShardSpec.scala

```diff
@@ -47,6 +58,7 @@ class PersistentShardSpec extends AkkaSpec(PersistentShardSpec.config) with AnyW
       system.log.info("Starting shard again")
       val secondIncarnation = system.actorOf(props)
 
+      // FIXME how did this ever work when the RestartEntity goes through the parent, expected to be a shard region?
```

Comment:
Explanation: In PersistentShard/master the GetShardStats is responded with the entity ids from the state even if they are not yet started when the query comes in.